file-formats/susi/susi-v1.json

```
@@ -0,0 +1,216 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/susi/susi-v1.json",
+  "title": "SUSI Object Type",
+  "description": "Object type SUSI",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "ABAP File Format Version",
+      "description": "The ABAP file format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "general": {
```

This will be updated as soon as you change the ABAP type and regenerate the schema
```suggestion
    "generalInformation": {
```